### PR TITLE
[incubator][VC] add extend info (deployment) to vPod

### PR DESCRIPTION
**Pull request by zhuangqh**
Signed-off-by: zhuangqh <zhuangqhc@gmail.com>

### CODE REVIEW

File: incubator/virtualcluster/pkg/syncer/conversion/mutate.go [hunk 1]
```diff
@@ -256,12 +259,40 @@ func omitDuplicates(strs []string) []string {
 	return ret
 }
 
+// TODO(zhuangqh): make this plugable.
+func (p *podMutator) attachTenantMeta(vPod *v1.Pod) error {
+	if len(vPod.ObjectMeta.OwnerReferences) == 0 {
```
Comment (Fei-Guo) [inline incubator/virtualcluster/pkg/syncer/conversion/mutate.go]: You cannot assume all Pods in tenant are deployment Pods. The Pods can be standard alone Pods and owned by any controller.

File: incubator/virtualcluster/pkg/syncer/conversion/mutate.go [hunk 2]
```diff
@@ -256,12 +259,40 @@ func omitDuplicates(strs []string) []string {
 	return ret
 }
 
+// TODO(zhuangqh): make this plugable.
```
Comment (Fei-Guo) [inline incubator/virtualcluster/pkg/syncer/conversion/mutate.go]: You can add a comment saying that for now, only Deployment Pods are mutated.

File: incubator/virtualcluster/pkg/syncer/conversion/mutate.go (hunk 1 above)
Comment (zhuangqh) [reply incubator/virtualcluster/pkg/syncer/conversion/mutate.go]: done

**Pull request merged**